lib/config/config-file.js

```diff
@@ -313,7 +313,7 @@ function write(config, filePath) {
 function getBaseDir(configFilePath) {
 
     // calculates the path of the project including ESLint as dependency
-    var projectPath = path.resolve(__dirname, "../../../");
+    var projectPath = path.resolve(__dirname, "../../../../");
```

Comment:
Because it's wrong.
Do you think this `basedir` and `lookupPath` are correct?

```
load( C:\Users\t-nagashima.AD\Documents\GitHub\sandbox\.eslintrc.json undefined undefined )
    resolvedPath: { filePath: 'C:\\Users\\t-nagashima.AD\\Documents\\GitHub\\sandbox\\.eslintrc.json' }
    dirname: C:\Users\t-nagashima.AD\Documents\GitHub\sandbox
    basedir: C:\Users\t-nagashima.AD\Documents\GitHub\sandbox\node_modules
    lookupPath: C:\Users\t-nagashima.AD\Documents\GitHub\sandbox\node_modules\node_modules
```